### DISCUSSION TITLE
How to perform feature branching in concourse 

### DISCUSSION THREAD

**Discussion by abhipsnl** [concourse/concourse#5395] [General]
I tried alot, couldn't find a answer. 
Is there a way in concourse to get feature branch for git resource?

branch: master      works 
branch: dev             works. 
branch:  feature/*   ??

I want whenever user create a feature branch , build should get triggered

**Comment by vito** [comment]
This is not yet natively supported; it is the primary focus of the so-called [v10 roadmap](https://blog.concourse-ci.org/core-roadmap-towards-v10/).

Work on this is under way - the `set_pipeline` step has been implemented, and we're now working on [pipeline archiving](https://github.com/concourse/concourse/projects/60). After that we'll need [instanced pipelines](https://github.com/concourse/rfcs/pull/34), the [`across` step](https://github.com/concourse/rfcs/pull/29), and some form of the [prototypes proposal](https://github.com/concourse/rfcs/pull/37) so that folks can write prototypes which represent branches and other things to automate.

For more information there's also the [Re-inventing resource types](https://blog.concourse-ci.org/reinventing-resource-types/) blog post, which provides some background on the "prototypes" concept.
